R/utils.R

```diff
@@ -19,17 +23,29 @@
 #' )
 #' assay(se)
 #' assay(relativeAb(se))
+#' 
+#' 
+#' utility function for relativeAb to return counts per million
+#'
+#' @description
+#' This function calculates the relative abundance of each feature in a dataset,
+#' expressed as counts per million (CPM) 
+#' 
+#' @param Logical value indicating whether counts should be normalized
```

Comment:
What param (argument) is being documented on line 34?

Reply:
I just updated my the code by adding se as the param (argument)

I think the se parameter is the primary parameter controlling the behavior of this utility function. like, it returns a new SummarizedExperiment object with counts per million calculated and added as a new assay named rel_abs.